cmd/fetchd/cmd/genesis-asi-upgrade.go

```diff
@@ -50,6 +50,10 @@ var networkInfos = map[string]NetworkConfig{
 			NewDenom:     "aasi",
 			OldDenom:     "afet",
 		},
+		SupplyInfo: SupplyInfo{
+			AdditionalSupplyValue:     "100000000000000000000000000",                  // TODO(JS): likely amend this
+			UpdatedSupplyOverflowAddr: "fetch15p3rl5aavw9rtu86tna5lgxfkz67zzr6ed4yhw", // TODO(JS): likely amend this
```

Comment:
If possible, I would suggest to use different term `overflow`, something like:
```suggestion
			DestinationWalletForNewlyMintedSupply: "fetch15p3rl5aavw9rtu86tna5lgxfkz67zzr6ed4yhw", // TODO(JS): likely amend this
```